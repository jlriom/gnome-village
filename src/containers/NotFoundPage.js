import React from "react";
import { Link } from "react-router-dom";
import PageLayout from "../components/layout/PageLayout";

const NotFoundPage = () => {
	return (
		<PageLayout headerText='Page not found'>
			<Link to='/'>Go to Home page</Link>
		</PageLayout>
	);
};

export default NotFoundPage;
