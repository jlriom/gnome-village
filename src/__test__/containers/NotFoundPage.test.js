import React from "react";
import NotFoundPage from "../../containers/NotFoundPage";
import renderer from "react-test-renderer";
import { MemoryRouter } from 'react-router-dom';

describe("<NotFoundPage />", () => {
    it("renders ok", () => {
        const tree = renderer.create(
            <MemoryRouter>
                <NotFoundPage />
            </MemoryRouter>
        );
        expect(tree).toMatchSnapshot();
    });
});

