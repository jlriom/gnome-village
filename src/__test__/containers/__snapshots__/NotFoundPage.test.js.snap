// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> renders ok 1`] = `
Array [
  <h2
    className="my-3 display-4"
  >
    Page not found
  </h2>,
  <hr />,
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to Home page
    </a>
  </div>,
]
`;
